squadcast/data_source_squadcast_escalation_policy.go

```diff
@@ -0,0 +1,75 @@
+package squadcast
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-provider-squadcast/types"
+)
+
+const escalationPolicyPath string = "/escalation-policies"
+
+var escalationPolicyRes types.EscalationPolicyRes
+
+func dataSourceSquadcastEscalationPolicy() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceSquadcastEscalationPolicyRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				// Sensitive:   true,
+				Description: "Escalation policy name",
+			},
+			"id": {
+				Type:        schema.TypeString,
+				Description: "ObjectId of the escalation policy",
+				Optional:    true,
+			},
+		},
+	}
+}
+
+func dataSourceSquadcastEscalationPolicyRead(resourceData *schema.ResourceData, configMetaData interface{}) error {
+	var escalationPolicyName = resourceData.Get("name").(string)
+	var squadcastConfig = configMetaData.(Config)
+
+	if squadcastConfig.AccessToken != "" {
+		log.Printf("[INFO] Access token is not set")
```

Comment:
shouldn't this be error instead of info?

Reply:
yeah, made the changes